fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/ReorgTable.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.common;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import com.ibm.fhir.database.utils.api.IDatabaseStatement;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+
+/**
+ * Reorg the schema.table
+ */
+public class ReorgTable implements IDatabaseStatement {
+    private final String schemaName;
+    private final String tableName;
+
+    /**
+     * Public constructor
+     * @param schemaName
+     * @param tableName
+     */
+    public ReorgTable(String schemaName, String tableName) {
+        DataDefinitionUtil.assertValidName(schemaName);
+        DataDefinitionUtil.assertValidName(tableName);
+        this.schemaName = schemaName;
+        this.tableName = tableName;
+    }
+
+    @Override
+    public void run(IDatabaseTranslator translator, Connection c) {
+        final String qname = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
+        final String ddl = "CALL SYSPROC.ADMIN_CMD ('REORG TABLE " + qname + "')";
```

Comment:
I think you need to run this... https://dba.stackexchange.com/a/30233/202822
I think this should be a second run.

Reply:
that answer is from 2012, i wonder if its still the "best practice".  it even mentions that the first runstats is more for older version...

Reply:
```RUNSTATS
REORG
RUNSTATS
REBIND packages
FLUSH PACKAGE CACHE DYNAMIC```
i wonder how long all that takes?
at least with db2 on cloud, it should be running runstats automatically, although maybe its still "best practice" to run it before and after the reorg...will add it for now

Reply:
on the dataset we had it took 10s of minutes in total

Reply:
I added the requested logic in `Db2Adapter.runStatement`...but I only did the RUNSTATS sandwhich and not the rebind/flush for now.
Then, later I decided that maybe the whole REORG statement is db2-specific, so I renamed the class to Db2ReorgTable (so I guess I could have had the logic here all-along...).